tests/ci/run_fips_callback_tests.sh

```diff
@@ -70,19 +70,31 @@ function run_all_break_tests() {
 }
 
 echo "Testing AWS-LC static breakable build with custom callback and Jitter enabled"
-build_and_test -DFIPS=1 \
-  -DCMAKE_C_FLAGS="-DBORINGSSL_FIPS_BREAK_TESTS -DAWSLC_FIPS_FAILURE_CALLBACK" \
+build_and_test -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_SHARED_LIBS=OFF \
+  -DFIPS=1 \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_INSTALL_INCLUDEDIR=include \
+  -DCMAKE_INSTALL_BINDIR=bin \
+  -DCMAKE_C_FLAGS="-ggdb -DBORINGSSL_FIPS_BREAK_TESTS -DAWSLC_FIPS_FAILURE_CALLBACK" \
   -DCMAKE_CXX_FLAGS="-DAWSLC_FIPS_FAILURE_CALLBACK" \
+  -DBUILD_TESTING=ON -DBUILD_LIBSSL=ON \
   -DENABLE_FIPS_ENTROPY_CPU_JITTER=1
 
 maybe_run_fips_tests
 maybe_run_fips_break_tests
 run_all_break_tests
 
 echo "Testing AWS-LC static build with custom callback and Jitter enabled"
-build_and_test -DFIPS=1 \
-  -DCMAKE_C_FLAGS="-DAWSLC_FIPS_FAILURE_CALLBACK" \
+build_and_test -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_SHARED_LIBS=OFF \
+  -DFIPS=1 \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_INSTALL_INCLUDEDIR=include \
+  -DCMAKE_INSTALL_BINDIR=bin \
+  -DCMAKE_C_FLAGS="-ggdb -DAWSLC_FIPS_FAILURE_CALLBACK" \
```

Comment:
Since you're enabling debug should the `CMAKE_BUILD_TYPE` instead be "RelWithDebInfo"?

Reply:
I was following our internal pattern where regardless of the build type we pass in `-ggdb`